bin/convert_finder_schemas

```diff
@@ -34,7 +33,16 @@ class DocumentTypeMapper
   }
 
   def call(filename)
-    MAPPING.fetch(File.basename(filename))
+    file_name = File.basename(filename)
+    begin
+      MAPPING.fetch(file_name)
+    rescue Exception => e
```

Comment:
I agree with @danielroseman.

As an aside, rescuing `Exception` is bad practice as it is the root class of the error class hierarchy, so this code would rescue things like `SystemExit`. Rescuing `KeyError` would be the way to go.